A32NX/html_ui/Pages/A32NX_Utils/NXSpeeds.js

```diff
@@ -492,6 +492,28 @@ class NXToSpeeds {
     }
 }
 
+class NXApprSpeeds {
+    /**
+     * Calculates VLS and Vapp for selected landing configuration
+     * @param {number} m Projected landing mass in t
+     * @param {boolean} isConf3 CONF3 if true, else FULL
+     * @param {number} [wind=live measured] tower headwind component
+     */
+    constructor(m, isConf3, wind = NaN) {
+        const cm = _correctMass(m);
+        this.vls = vls[isConf3 ? 3 : 4][cm](m, 1);
+        if (isFinite(wind)) {
+            this.vapp = this.vls + NXSpeedsUtils.addWindComponent(wind / 3);
+        } else {
+            this.vapp = this.vls + NXSpeedsUtils.addWindComponent();
+        }
```

Comment:
consider using parameter default

```suggestion
    constructor(m, isConf3, wind = SimVar.GetSimVarValue("AIRCRAFT WIND Z", "knots") * -1) {
        const cm = _correctMass(m);
        this.vls = vls[isConf3 ? 3 : 4][cm](m, 1);
        this.vapp = this.vls + _addWindComponent(wind / 3);
```